src/module-analyzer.js

```diff
@@ -274,6 +275,9 @@ export class ModuleAnalyzer {
         } else if (conventional = BindingBehaviorResource.convention(key)) {
           resources.push(new ResourceDescription(key, exportedValue, conventional));
           metadata.define(metadata.resource, conventional, exportedValue);
+        } else if (conventional = ViewEngineHooksResource.convention(key)) {
```

Comment:
Sort of the same reason as above. This also applies to the existing code for `ValueConverterResource` and `BindingBehaviorResource`, so I just followed the convention.